pkg/abaputils/abaputils.go

```diff
@@ -51,8 +51,10 @@ func (abaputils *AbapUtils) GetAbapCommunicationArrangementInfo(options AbapEnvi
 		var hostOdataURL = options.Host + oDataURL
 		if match {
 			connectionDetails.URL = hostOdataURL
+			connectionDetails.Host = options.Host
 		} else {
 			connectionDetails.URL = "https://" + hostOdataURL
+			connectionDetails.Host = "https://" + options.Host
 		}
```

Comment:
Does it make sense to have both `URL` and `Host` parameters set here?

Reply:
The URL is extended with the path at different spots throughout the abap functions. This could be a candidate for a later refactoring, but would require more effort